### PR TITLE
UHM-3588 - added chart to lab trends page

### PR DESCRIPTION
**Pull request by rhenshaw56**

### CODE REVIEW

File: app/js/components/LabTrendsPage.jsx [hunk 1]
```diff
@@ -93,19 +125,33 @@ export class LabTrendsPage extends PureComponent {
...
+              defaultPageSize={defaultPageSize}
+            />
+          </div>
+          <div className={chartClassName}>
+            <LineChart
+              data={formatChartData(R.reverse(results))}
+              height={280}
+              width={500}
+              yAxisLabel="units"
+              xAxisLabel="rowland"
```
Comment (jomadoye) [inline app/js/components/LabTrendsPage.jsx]: when did you become a label? :-)

File: app/js/components/LabTrendsPage.jsx [hunk 2]
```diff
@@ -93,19 +125,33 @@ export class LabTrendsPage extends PureComponent {
...
+              xAxisKey="obsDatetime"
+              yAxisKey="value"
+              type="linear"
+            />
+          </div>
         </div>
         <br />
-        <button type="button" className="btn btn-lg btn-danger" onClick={() => this.handleNavigateBack(history)}>Back</button>
+        <button type="button" className="btn btn-lg btn-danger" onClick={() => this.handleNavigateBack()}>Back</button>
```
Comment (jomadoye) [inline app/js/components/LabTrendsPage.jsx]: `this.handleNavigateBack`

File: app/js/components/LabTrendsPage.jsx [hunk 3]
```diff
@@ -75,8 +83,32 @@ export class LabTrendsPage extends PureComponent {
...
+    });
+
+    const chartClassName = cn({
+      'lab-results-trend-chart': true,
+      hide: !isNumeric,
+    });
+
     this.setState({ defaultPageSize: calculateTableRows(results.length) });
 
+    const formatChartData = data => data.map(item => ({
```
Comment (jomadoye) [inline app/js/components/LabTrendsPage.jsx]: Is it guaranteed that the data would always be sorted by date? If not this would be a good place to sort the data by date. because it seems reChart doesn't organize the data, it basically renders what it gets as is.

File: app/js/components/LabTrendsPage.jsx [hunk 4]
```diff
@@ -20,7 +25,7 @@ import {
   getSampleDate, getResultValue, calculateTableRows,
 } from '../utils/helpers';
 
-import "../../css/lab-orders-trends-page.scss";
+import "../../css/lab-results-trends-page.scss";
```
Comment (jomadoye) [inline app/js/components/LabTrendsPage.jsx]: The new file is not part of this PR,

File: app/js/components/LabTrendsPage.jsx (hunk 3 above)
Comment (mogoodrich) [reply app/js/components/LabTrendsPage.jsx]: Agreed, we should sort by date to be safe
